src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
```

Comment:
I'm a little confused too, so maybe some REM's in the script will help explain.  Am I correct in assuming you expect this CMD to live in the git repo but be executed from some location other than the git repo folder?  I can understand how that might be required for a 'git pull' to succeed when this script changes.  It also means (I think) that any changes to this script need to be manually copied to a different location on the machine.

Here's another approach to consider.  Create a script that gets copied to %temp% and is started there.  That script could then do the 'git clean' and 'git pull' of the repo (without needing a %2 to know where it lives) and then calls back into the "real" launch script living in the repo.  That would allow the 'git pull' to retrieve a new copy of the "real" launch script and would not require us to manually maintain a separate folder.